data/sources/01_areas-extended.yaml

```diff
@@ -681,6 +728,73 @@ garching:
   image_comments:
     1: "Orthographische Luftbildaufnahme des Campus Garching"
 
+# --- Buildings not in TUMonline ---
+ipp:
+  coords: { lat: 48.26315, lon: 11.67206 }
+  osm: ["relation/8605192"]
+  props:
+    address:
+      street: "Boltzmannstraße 2"
+      plz_place: "85748 Garching bei München"
+      source: "navigatum"
+
+mpa:
+  coords: { lat: 48.26099, lon: 11.67143 }
+  osm: ["relation/3315967"]
+  props:
+    address:
+      street: "Karl-Schwarzschild-Straße 1"
+      plz_place: "85748 Garching bei München"
+      source: "navigatum"
+
+mpb:
+  coords: { lat: 48.10532, lon: 11.46073 }
+  osm: ["way/28704007", "way/28704014", "way/200674891", "way/54690213"]
+  props:
+    address:
+      street: "Am Klopferspitz 18"
+      plz_place: "82152 Martinsried"
+      source: "navigatum"
+
+mpe:
+  coords: { lat: 48.26165, lon: 11.67180 }
+  osm: ["way/28260009"]
+  props:
+    address:
+      street: "Gießenbachstraße 1"
+      plz_place: "85748 Garching bei München"
+      source: "navigatum"
+
+mpp:
+  coords: { lat: 48.18469, lon: 11.61263 }
+  osm: ["way/32134827", "way/24052546"]
+  props:
+    address:
+      street: "Föhringer Ring 6"
+      plz_place: "80805 München"
+      source: "navigatum"
+
+mpq:
+  coords: { lat: 48.25962, lon: 11.66682 }
+  osm: ["way/172773645"]
+  props:
+    address:
+      street: "Hans-Kopfermann-Straße 1"
+      plz_place: "85748 Garching bei München"
+      source: "navigatum"
+
+origins-cluster:
+  coords: { lat: 48.26390, lon: 11.67367 }
+  osm: ["way/159713052"]
+  props:
+    address:
+      # Official address, even though not exactly where the building is
```

Comment:
Should we add this as a comment in the frontend, or do you think this is probably fine as is?
I also could not get a definitive answer

Reply:
The coordinates in the NAT Roomfinder do at the moment point to the building that we have selected and according to OSM it's also the one. I would leave it like this I think because I am not sure enough about how it's intended to be to write something in the Frontend. We could merge now but ask someone from there at the same time and fix if needed.